bake/bake.go

```diff
@@ -256,24 +272,23 @@ func ParseFile(dt []byte, fn string) (*Config, error) {
 	return ParseFiles([]File{{Data: dt, Name: fn}}, nil)
 }
 
-func ParseComposeFile(dt []byte, fn string) (*Config, bool, error) {
+func ParseComposeFile(dt []byte, fn string) (bool, error) {
```

Comment:
I like the idea of keeping a smaller footprint for this patch, but as this doesn't return a `Config` struct anymore, I think this might make more sense as `Validate`/etc instead of `Parse`?

May also be worth keeping as a private function IMO.